src/ioq_server2.erl

```diff
@@ -959,6 +959,7 @@ cleanup(Servers) ->
 
 instantiate(S) ->
     Old = ?DEFAULT_CONCURRENCY * length(ioq_sup:get_ioq2_servers()),
+    ?debugFmt(" +++ ioq_sup:get_ioq2_servers():~p~n~n", [ioq_sup:get_ioq2_servers()]),
```

Comment:
Forgot debug statement

Reply:
I changed ["DEFAULT_CONCURRENCY" from 1 to 9](https://github.com/apache/couchdb-ioq/pull/20/files#diff-d61812c4ac5fd46df0ed136693304f4a07a1b96784a039f67cb0e7085650586dR47) to pass eunit tests, I'm not too sure about this.